server/services/userService.ts

```diff
@@ -1,25 +1,28 @@
 import { convertToTitleCase } from '../utils/utils'
-import type HmppsAuthClient from '../data/hmppsAuthClient'
-import type { User } from '../data/hmppsAuthClient'
-import { NomisUserRolesApi, type UserCaseloads } from '../data/nomisUserRolesApi'
+import { NomisUserRolesApi, type Caseload } from '../data/nomisUserRolesApi'
+import ManageUsersApiClient, { User } from '../data/manageUsersApiClient'
 
-export interface UserDetails extends User, UserCaseloads {
+export interface UserDetails extends User {
   displayName: string
+  caseloads: Array<Caseload>
+  activeCaseload: Caseload
 }
```

Comment:
i think this type was better represented by what we had here before cos `UserDetails` is made by merging a `User` object and a `UserCaseloads` object and adding a new "displayName" field. so that means if `UserCaseloads` changes, this `UserDetails` type would automatically inherit the change without needing to change this definition

Reply:
Ah yeah, good point. Ty 👍